lib/active_merchant/billing/gateways/litle.rb

```diff
@@ -39,6 +39,88 @@ def purchase(money, payment_method, options = {})
         check?(payment_method) ? commit(:echeckSales, request, money) : commit(:sale, request, money)
       end
 
+      def add_level_2_data(doc, payment_method, options = {})
```

Comment:
💬 
you can change the name to add_level_two_data and and_level_three_data instead of use numbers in the method name.

Reply:
thanks for the suggestion, updated!